File: ApplicationCore/ComicBase.Core/Domain/Entities/Publisher.cs
using ComicBase.Core.Domain.Interfaces;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace ComicBase.Core.Domain.Entities
{
    /// <summary>
    /// A comic book publisher entity.
    /// </summary>
    public class Publisher : EntityBase
    {
        [Required]
        public string Name { get; set; }

        public string SiteUrl { get; set; }
    }
}

File: UserInterfaces/UI.Aurelia/ClientApp/app/components/Publisher/Models/Publisher.ts
export class Publisher {

    constructor(id: number, name: string, siteUrl: string) {

        this.Id = id;
        this.Name = name;
        this.SiteUrl = siteUrl;

    }

    public Id?: number;
    public Name: string;
    public SiteUrl: string;

}